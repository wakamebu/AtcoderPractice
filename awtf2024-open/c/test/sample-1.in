5
1 10 4
7
1
1 10 6
7
1
2 12 3
5 7
1 1
2 12 3
5 7
1 2
20 749013197 23809523
46981984 70791437 118235723 132421762 180040807 203849360 251468335 275277857 322889975 346699150 394318091 418113855 465732891 489532137 537144103 558852533 606466719 630275002 677584754 701394209
1 2 2 1 1 2 1 2 1 2 2 1 2 1 2 1 1 2 1 2
