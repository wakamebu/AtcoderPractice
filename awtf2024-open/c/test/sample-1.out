2
0
-1
6
585545066743659
